cpp/CMake/FindMlpack.cmake

```diff
@@ -0,0 +1,14 @@
+# Store the mlpack include directory in MLPACK_INCLUDE_DIR.
```

Comment:
Can we remove the file here? I think the right place is probably the mlpack model's repository.